.github/workflows/deploy_to_environment_azure.yml

```diff
@@ -0,0 +1,171 @@
+name: Deploy to environment (CIP)
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment (staging, production)"
+        required: true
+
+env:
+  DOCKER_IMAGE: a2bint-app
```

Comment:
Would it be alright to rename to conversions-app?
We want to move away from the name a2b internal eventually.

Reply:
Yep that shouldn't be a problem. The only thing we'd need to update is the reference to the [image name in Terraform](https://github.com/DFE-Digital/a2b-internal/blob/main/terraform.gpaas-azure-migration/terraform.tfvars.example#L5)